pages/property-filter/common-props.tsx

```diff
@@ -178,19 +180,57 @@ export const i18nStrings: PropertyFilterProps.I18nStrings & I18nStringsExt = {
   tokenLimitShowFewer: 'Show fewer',
   clearFiltersText: 'Clear filters',
   tokenOperatorAriaLabel: 'Boolean Operator',
-  removeTokenButtonAriaLabel: () => 'Remove token',
   enteredTextLabel: (text: string) => `Use: "${text}"`,
 
-  tokenEditorTokenActionsLabel: token =>
-    `Filter remove actions for ${token.propertyLabel} ${token.operator} ${token.value}`,
-  tokenEditorTokenRemoveLabel: () => 'Remove filter',
-  tokenEditorTokenRemoveFromGroupLabel: () => 'Remove filter from group',
+  formatToken,
+  removeTokenButtonAriaLabel: token => `Remove token, ${formatToken(token)}`,
+
+  groupEditAriaLabel: group => `Edit group with ${group.tokens.length} tokens`,
+  tokenEditorTokenActionsAriaLabel: token => `Filter remove actions for ${formatToken(token)}`,
+  tokenEditorTokenRemoveAriaLabel: token => `Remove filter, ${formatToken(token)}`,
+  tokenEditorTokenRemoveLabel: 'Remove filter',
+  tokenEditorTokenRemoveFromGroupLabel: 'Remove filter from group',
   tokenEditorAddNewTokenLabel: 'Add new filter',
-  tokenEditorAddTokenActionsLabel: 'Add filter actions',
-  tokenEditorAddExistingTokenLabel: token =>
-    `Add filter ${token.propertyLabel} ${token.operator} ${token.value} to group`,
+  tokenEditorAddTokenActionsAriaLabel: 'Add filter actions',
+  tokenEditorAddExistingTokenAriaLabel: token => `Add filter ${formatToken(token)} to group`,
+  tokenEditorAddExistingTokenLabel: token => `Add filter ${getTokenLabel(token)} to group`,
 };
 
+function getTokenLabel(token: PropertyFilterProps.FormattedToken) {
+  return `${token.propertyLabel} ${token.operator} ${token.value}`;
+}
+
+function formatToken(token: PropertyFilterProps.FormattedToken) {
+  return `${token.propertyLabel} ${operatorToLabel(token.operator)} ${token.value}`;
+}
+
+function operatorToLabel(operator: string) {
```

Comment:
can't we re-use `operatorToDescription` from [i18n-utils](https://github.com/cloudscape-design/components/blob/3a824d0850f46c462ea622a7d54881facb8cad58/src/property-filter/i18n-utils.ts#L219)?

Reply:
We can, but that one is meant to be internal while the pages serve as an examples of what the customer would do if they don't use the built-in i18n.